sw/simulator/pprzsim-launch

```diff
@@ -78,6 +78,8 @@ def main():
                         help="Number of RC script to use")
     nps_opts.add_option("--time_factor", type="float", action="store", metavar="factor",
                         help="Time factor (default 1.0)")
+    nps_opts.add_option("--fg_fdm", type="int", action="store", metavar="factor",
```

Comment:
no reason to require an arg here... so `"--fg_fdm", action="store_true",` would be nicer